server/api/categoryRouter.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Router for item actions
```

Comment:
Should this be "Router for category actions?"

Reply:
yes!!